addons/mail/static/src/discuss/core/public_web/discuss_sidebar_categories.scss

```diff
@@ -42,7 +42,7 @@
 }
 
 .o-mail-DiscussSidebarCategory-foldIndicatorCompact {
-    left: 6px;
+    left: -3px;
```

Comment:
Wouldn't it be ok with 0? I don't like values too close to border of window, and it feels too far from the main icon also.

But I think the chevron is too much visual info in compact mode actually. Double icons isn't great, but I don't know what else we could do.

Reply:
We still need it, but probably we could play with its sizing and/or opacity. I suppose if this doesn't look good now is due to chevron being too visible

Reply:
I put `start-0` in the end